src/components/NavBar.tsx

```diff
@@ -42,15 +43,21 @@ export const Navbar = ({ handleOpenDrawer }: NavbarProps) => {
         <Menu.Item
           className='navbar-menu-item'
           key='/dashboard'
-          icon={<DollarTwoTone twoToneColor='#4c88ef' />}>
+          icon={<DollarTwoTone twoToneColor={twoToneColor} />}>
```

Comment:
`twoToneColor` is not descriptive, please rename it to a descriptive/distinguishable colour name